Dockerfile

```
@@ -22,8 +23,23 @@ RUN apt-get install -y --no-install-recommends npm
 RUN python3 -m pip install -e . jupyter-packaging
 RUN python3 setup.py bdist_wheel --dist-dir /voila/dist
 
-
 FROM base_image_python as voila_rootfs_builder
+# Miniconda
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
+ARG mconda="miniconda-installer.sh"
+# Pin to 4.14.0, Python 3.7, Linux64
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-py37_4.12.0-Linux-x86_64.sh -O $mconda && \
+    bash $mconda -b && \
+    rm -f $mconda
+# Create conda env
+COPY resilience-env.yml .
+RUN conda env create -n voilaenv --file resilience-env.yml
+# Use conda env for subsequent RUNs
+RUN echo "conda activate myenv" >> ~/.bashrc
```

I will fix or eliminate altogether.
```suggestion
RUN echo "conda activate voilaenv" >> ~/.bashrc
```